webpack.dev.config.babel.js

```diff
@@ -128,4 +126,15 @@ export default Object.assign({}, webpackConfig, {
     new webpack.IgnorePlugin(/webpack-stats\.json$/),
     webpackIsomorphicToolsPlugin.development(),
   ],
+  resolve: {
```

Comment:
Actually yes, sorry this was a holder from testing and should be removed, thanks!